src/Component/DependencyInjection/src/Argument/ServiceClousureArgument.js

```diff
@@ -0,0 +1,35 @@
+const ArgumentInterface = Jymfony.Component.EventDispatcher.ArgumentInterface;
+const Reference = Jymfony.Component.DependencyInjection.Reference;
+
+class ServiceClousureArgument extends implementationOf(ArgumentInterface) {
+
+    /**
+     * @param {Jymfony.Component.DependencyInjection.Reference} reference
+     * @private
+     */
+    __construct(reference) {
+        this._values = [ reference ];
+    }
+
+    /**
+     *
+     * @return {Array|*}
```

Comment:
Correct annotation is `@returns {[*]}`. Please fix all the `@return` adding the missing "s"